lsp/test/JsonSpec.hs

```diff
@@ -7,6 +7,9 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 -- For the use of MarkedString
 {-# OPTIONS_GHC -fno-warn-deprecations #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
More pragmas with no module changes, maybe they are needed for upstream changes?